app/controllers/people_controller.rb

```diff
@@ -47,6 +47,9 @@ def destroy
   private
 
   def person_params
-    params.require(:person).permit(:first_name, :last_name, :email, :password, :password_confirmation, :date_of_birth, :gender)
+    params.require(:person).permit(
+      :first_name, :last_name, :email, :password,
+      :password_confirmation, :date_of_birth, :gender
```

Comment:
Haha I actually agree with this :P I didnt actually review the Rubocop corrections :(